fe/fe-core/src/main/java/org/apache/doris/common/FeNameFormat.java

```diff
@@ -138,6 +140,12 @@ public static void checkCommonName(String type, String name) throws AnalysisExce
         }
     }
 
+    public static void checkSuccessFileName(String type, String name) throws AnalysisException {
```

Comment:
```suggestion
    public static void checkOutfileSuccessFileName(String type, String name) throws AnalysisException {
```
May it is better to call this method as `checkOutfileSuccessFileName`